Assets/Scripts/Utils/VectorUtils.cs

```diff
@@ -172,6 +172,24 @@ public static float Max(this Vector3 v)
         {
             return Mathf.Max(v.x, v.y, v.z);
         }
+
+        public static Vector2 Rotate(this Vector2 vector, float degrees)
+        {
+            return new Vector2(
+                vector.x * Mathf.Cos(degrees * Mathf.Deg2Rad) - vector.y * Mathf.Sin(degrees * Mathf.Deg2Rad),
+                vector.x * Mathf.Sin(degrees * Mathf.Deg2Rad) + vector.y * Mathf.Cos(degrees * Mathf.Deg2Rad)
+            );
+        }
+
+        public static string ToPrint(this Vector2 vector)
```

Comment:
Are these any different from the built-in Vector2.ToString() and Vector3.ToString() functions?